packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -424,6 +424,44 @@ export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
       }
     }
 
+    // 8: generate launch configuration
+    channelService.appendLine(
+      nls.localize('isv_debug_bootstrap_generate_launchjson')
+    );
+    try {
+      const projectVsCodeFolder = path.join(projectPath, '.sfdx');
+      shell.mkdir('-p', projectVsCodeFolder);
+      fs.writeFileSync(
+        path.join(projectVsCodeFolder, 'launch.json'),
+        // mostly duplicated from ApexDebuggerConfigurationProvider to avoid hard dependency from core to debugger module
+        JSON.stringify({
+          version: '0.2.0',
+          configurations: [
+            {
+              name: 'Launch Apex Debugger',
+              type: 'apex',
+              request: 'launch',
+              userIdFilter: [],
+              requestTypeFilter: [],
+              entryPointFilter: '',
+              sfdxProject: '${workspaceRoot}',
+              connectType: 'ISV_DEBUGGER'
+            }
+          ]
+        }),
+        { encoding: 'utf-8' }
```

Comment:
Nitpick: This can be just `'utf-8'` because `writeFileSync` has an overload whose second param is a string to specify the encoding. I've seen uses of `fs` in our other extensions follow that approach.